app/models/project.rb

```diff
@@ -86,4 +86,8 @@ def score(nickname, token)
   def contribulator_url
     "https://contribulator.24pullrequests.com/#{github_repository}"
   end
+
+  def url
+    homepage || github_url
```

Comment:
Given this is only exposed to admin users at the moment, more validation probably isn't required?